configs/swin/swin_mask_rcnn/mask_rcnn_swim-t-p4-w7_fpn_fp16_ms-crop-3x_coco.py

```diff
@@ -0,0 +1,3 @@
+_base_ = './mask_rcnn_swim-t-p4-w7_fpn_ms-crop-3x_coco.py'
+# you need to set mode='dynamic' if you are using pytorch<=1.5.0
+fp16 = dict(loss_scale=dict(init_scale=512))
```

Comment:
Could you provide reproducing information about fp16 in `swin/README.md`?
[The authors adopt apex instead of mmdet's fp16 for stability](https://github.com/SwinTransformer/Swin-Transformer-Object-Detection/issues/1), but details are unknown.

Reply:
MIM examples have already tried that. The results is stable and compatible https://github.com/open-mmlab/mim-example/tree/master/swin_transformer

Reply:
Thank you! I didn't realize that the CVPR tutorial files were already published.